js/carousel.js

```diff
@@ -0,0 +1,21 @@
+document.addEventListener("DOMContentLoaded", function () {
+  var elem = document.querySelector(".js-flickity");
+  var flkty = new Flickity(elem, {
+    freeScroll: true,
+    wrapAround: true,
+    prevNextButtons: false,
+    pageDots: false,
+  });
+
+  document
+    .querySelector(".carousel-control-prev-custom")
```

Comment:
Perhaps you can reserve a prefixed class name here as well? Like You did at line 2 with "js-", for example "js-carousel-previous" and. "js-carousel-next".